src/RaisedButton/RaisedButton.spec.js

```diff
@@ -91,4 +91,16 @@ describe('<RaisedButton />', () => {
       );
     });
   });
+
+  it('inherits fontSize from theme', () => {
+    const wrapper = mountWithContext(
```

Comment:
Unfortunately not. Shallow rendering doesn't go deep enough to render the label.